packages/plugin-lighthouse/src/lib/runner/utils.unit.test.ts

```diff
@@ -126,14 +128,45 @@ describe('toAuditOutputs', () => {
     );
   });
 
+  it('should not parse given audit details', () => {
+    expect(
+      toAuditOutputs(
+        [
+          {
+            id: 'cumulative-layout-shift',
+            details: {
+              type: 'table',
+              headings: [{ key: 'number' }],
+              items: [{ number: 42 }],
+            },
+            score: 0,
+            numericValue: 0,
+            displayValue: '0 ms',
+          } as unknown as Result,
```

Comment:
There's always a better way than `as unknown as T` (part 1):

```suggestion
          {
            id: 'cumulative-layout-shift',
            details: {
              type: 'table',
              headings: [{ key: 'number' }] as Details.TableColumnHeading[],
              items: [{ number: 42 }],
            } as Details,
            score: 0,
            numericValue: 0,
            displayValue: '0 ms',
          } as Result,
```